pkgs/top-level/all-packages.nix

```diff
@@ -7302,10 +7302,12 @@ with pkgs;
 
   gradle_7-unwrapped = callPackage gradle-packages.gradle_7 { };
   gradle_8-unwrapped = callPackage gradle-packages.gradle_8 { };
+  gradle_9-unwrapped = callPackage gradle-packages.gradle_9 { };
```

Comment:
This is a [discouraged pattern](https://github.com/NixOS/nixpkgs/actions/runs/15957240101/job/45005136359?pr=421047#step:5:170) according to **Lint / nixpkgs-vet** -- hopefully this can be merged as is and that issue can be resolved later, but I have no idea what the policy is and when exceptions are granted.  (Maybe we open an issue on Gradle requesting a move to `by-name/gr` as a sacrifice to the approval/merge gods? Or maybe a follow-on PR that removes gradle_7 to show that we are not _increasing_ the usage of the "discouraged pattern"?)

Reply:
I spoke to Nixpkgs maintainers and committers more knowledgable than me (at Nix Vegas) and they said this should not be an issue.